Tests/NetworkServiceTests/NetworkErrorTests.swift

```diff
@@ -0,0 +1,19 @@
+import NetworkService
+import XCTest
+
+final class NetworkErrorTests: XCTestCase {
+    func testAllNetworkErrorsExistInAPI() {
+        XCTAssertTrue(NetworkError.allCases.contains(.authenticationError))
```

Comment:
Why just you haven't put the checks in one assertion:
```suggestion
let possibleNetworkErrors = NetworkError.allCases
let expectedNetworkErrors = Set(A, B, C)

XCTAssertTrue(possibleNetworkErrors, expectedNetworkErrors)
```
?

Reply:
Thanks for the suggestion. Fixed it.